src/Compilers/CSharp/Portable/Binder/Semantics/OverloadResolution/OverloadResolution.cs

```diff
@@ -2566,11 +2566,11 @@ private BetterResult BetterFunctionMember<TMember>(
 
             // Returns the parameter type (considering params).
             static TypeSymbol getParameterTypeAndRefKind(int i, MemberAnalysisResult memberResolutionResult, ImmutableArray<ParameterSymbol> parameters,
-                TypeWithAnnotations paramsElementTypeOpt, TMember member, out RefKind parameter1RefKind)
+                TypeWithAnnotations paramsElementTypeOpt, TMember member, out RefKind parameterRefKind)
             {
                 ParameterSymbol parameter = getParameterOrExtensionParameter(i, memberResolutionResult, parameters, member);
 
-                parameter1RefKind = parameter.RefKind;
+                parameterRefKind = GetParameterBetternessRefKind(parameter, member);
```

Comment:
:memo: I don't think this is observable. The callers of this local function who care about the returned `parameterRefKind` are for scenarios where the receiver is a COM import type, but a COM import type must be class or interface, which can't be used for extension parameter with `ref` modifier.